backend/typescript/services/implementations/__tests__/schedulingService.test.ts

```diff
@@ -84,27 +87,22 @@ const testSchedules = [
     status: "Pending",
     volunteerNeeded: false,
     frequency: "Monthly",
+    recurringDonationId: RECURRING_DONATION_ID,
     recurringDonationEndDate: new Date("2021-10-01T00:06:00.000Z"),
     notes: "these are the copied notes",
   },
 ];
 
-const invalidTestSchedule = [
```

Comment:
this was never used so I deleted it but I can put it back in if this is still needed